options/options.h

```diff
@@ -44,6 +44,15 @@ enum Engine
   // used for setting solver options appropriately
 };
 
+// NOTE keep this up to date so that setting
```

Comment:
Please update also the comment in lines 42-44 above to refer to `ic3_variants_set` in `./options/options.cpp`, which must be updated when adding IC3 variants.

Reply:
Good catch! Fixed in e847cde3a673d90b0387600cf95b5ccee27b35ac.